Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 String cron_string = BRANCH_NAME == "master" ? "H 12 * * 1-5" : ""
 
 pipeline {
-  agent { label 'linux && !gpu' }
```

Comment:
The GCE jenkins plugins doesn't work with not (!) tags. But since we create them on the fly, we can ensure it spins up a VM w/o gpu. I will reach out to the maintainer of the plugin. I use to work with him.

Reply:
Nice to have those contacts 👍